Cargo.toml

```diff
@@ -54,7 +54,8 @@ async-std = { version = "1.5.0", optional = true}
 async-trait = "0.1.24"
 
 [features]
-default = ["streams", "geospatial", "tokio-comp", "async-std-comp", "script"]
+default = ["acl", "streams", "geospatial", "tokio-comp", "async-std-comp", "script"]
```

Comment:
I would remove `"acl"` from this list so that we can cope with the current CI environment.  Not an awesome suggestion, but I think having the ACL changes in the main branch are worth the compromise.